src/CodeGen_Internal.cpp

```diff
@@ -590,11 +590,9 @@ bool get_md_string(llvm::Metadata *value, std::string &result) {
     return false;
 }
 
-void get_target_options(const llvm::Module &module, llvm::TargetOptions &options, std::string &mcpu, std::string &mattrs) {
+void get_target_options(const llvm::Module &module, llvm::TargetOptions &options) {
     bool use_soft_float_abi = false;
     get_md_bool(module.getModuleFlag("halide_use_soft_float_abi"), use_soft_float_abi);
```

Comment:
To explicitly call this out: yes, none of this should be done here, ideally,
as much as possibly should be ported into `set_function_attributes_from_halide_target_options()`.

Reply:
I've filed https://github.com/halide/Halide/issues/6769 for now.